data-reconciliation-app/test/unit-test/services/ingest-service.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, expect, test, beforeEach, afterEach } from "@jest/globals";
+import * as jscrypto from "crypto";
```

Comment:
You don't have to use this at all. NodeJS has built in method for this :) 

```
import { randomUUID } from 'crypto';

const userId = randomUUID();
```

Remember to add following types to your `devDependencies`

```json
"@types/node": "^18.11.9",
```